include/blosc2/blosc2-stdio.h

```diff
@@ -45,7 +45,9 @@ BLOSC_EXPORT int blosc2_stdio_truncate(void *stream, int64_t size);
 
 
 /**
- * @brief Parameters for memory-mapped I/O.
+ * @brief Parameters for memory-mapped I/O. Existing schunks can be opened memory-mapped with the *_udio functions and
+ * new schunks be created by setting the io member of the #blosc2_storage struct (see test_mmap for examples). Please
```

Comment:
Not sure on what you mean here: mmap can be used in combination with schunks or the other way around?  A rephrasing would be good here.